packages/ember-intl/index.js

```diff
@@ -5,8 +5,6 @@
 const { existsSync } = require('node:fs');
 const { dirname, isAbsolute, join } = require('node:path');
 const mergeTrees = require('broccoli-merge-trees');
-const { BroccoliMergeFiles } = require('broccoli-merge-files');
-const stringify = require('json-stable-stringify');
```

Comment:
Note to myself:

We may be able to replace `json-stable-stringify` with `JSON.stringify()`.

It might be that `json-stable-stringify` had been introduced in order to alphabetize the keys of nested objects for tests. If the order doesn't matter for consuming projects, we can uninstall the library and update the test assertions accordingly.